pkgs/development/python-modules/vl-convert-python/default.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  rustPlatform,
+  protobuf,
+  libffi,
+  callPackage,
+  librusty_v8 ? callPackage ./librusty_v8.nix { },
+  nix-update-script,
+}:
+buildPythonPackage rec {
+  pname = "vl-convert-python";
+  version = "1.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "vega";
+    repo = "vl-convert";
+    tag = "v${version}";
+    hash = "sha256-dmfY05i5nCiM2felvBcSuVyY8G70HhpJP3KrRGQ7wq8=";
+  };
+
+  patches = [ ./libffi-sys-system-feature.patch ];
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-t952WH6zq7POVvdX3fI7kXXfPiaAXjfsvoqI/aq5Fn0=";
+  };
+
+  buildAndTestSubdir = "vl-convert-python";
+
+  env.RUSTY_V8_ARCHIVE = librusty_v8;
```

```suggestion
    env.RUSTY_V8_ARCHIVE = deno.RUSTY_V8_ARCHIVE;
```

is it possible to use rusty_v8 from deno ? i see there are already few copiles of rusty_v8 in nixpkgs, i wonder if it can be avoided

---

i guess the answer is no,  they all use different version of librusty_v8,